docs/kyma/docs/031-gs-local-installation.md

```diff
@@ -16,19 +16,17 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
 - [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
-- [Virtualbox](https://www.virtualbox.org/) - Linux or Windows
-- Hyper-V - Windows
+- [VirtualBox](https://www.virtualbox.org/) - Linux or Windows
+- [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/) - Windows
 
-Read the [prerequisite reasoning](019-prereq-reasoning.md) document to learn why Kyma uses these tools.
+## Set up certificates
 
-## Setup certificates
-
-Kyma comes with a local wildcard self-signed [certificate](../../../installation/certs/workspace/raw/server.crt). Trust it on the OS level for convenience. Alternatively, accept exceptions for each subdomain in your browser as you use Kyma.
+Kyma comes with a local wildcard self-signed `server.crt` certificate that you can find under the `/installation/certs/workspace/raw/` directory of the `kyma` repository. Trust it on the OS level for convenience. Alternatively, accept exceptions for each subdomain in your browser as you use Kyma.
 
 Follow these steps to "always trust" the Kyma certificate on macOS:
 
 1. Open the Keychain Access application. Select **System** from the **Keychains** menu.
-2. Go to **File**, select **Import items...**, and import the Kyma [certificate](../../../installation/certs/workspace/raw/server.crt).
+2. Go to **File**, select **Import items...**, and import the Kyma `server.crt` certificate.
```

Comment:
This instruction is replaced with a simpler one in 
https://github.com/kyma-project/kyma/pull/137

Reply:
Sure but I just got rid of the link here. You can paraphrase that later on however you wish.